qiskit_rng/generator.py

```diff
@@ -181,8 +181,7 @@ def _run_circuits(
                                        job_tags=[self._job_tag], memory=True)
             logger.info("Jobs submitted to %s. Job set ID is %s.", self.backend, job.job_set_id())
         else:
-            job = self.backend.run(assemble(transpiled, backend=self.backend, shots=shots,
-                                            memory=True, noise_model=self.noise_model))
+            job = self.backend.run(circuits, memory=True)
```

Comment:
You need to keep the same options that was specified in `assemble`, like `shots` etc